.lintstagedrc.json

```diff
@@ -1,6 +1,6 @@
 {
     "*.json": ["prettier --write", "git add"],
-    "*.php": ["npm run lint:textdomain", "npm run lint:php:fix", "git add"],
+    "*.php": ["npm run lint:textdomain", "npm run lint:php", "npm run lint:php:fix", "git add"],
```

Comment:
@bradp when I added `npm run lint:php`  then it was blocking us most of the time and few conditions were not easy to pass. I will suggest to remove it.

@JasonTheAdams any thought?

Reply:
@ravinderk I'm not sure what you're saying above. Can you expand so it's clear to me?

Reply:
![image](https://user-images.githubusercontent.com/1784821/82086572-4fa77e00-970c-11ea-9520-df8930033605.png)

@DevinWalker WPCS forces function naming with `_(underscore)` but now we started function naming in camel case also in `src` directory.
If we will autorun `npm run lint:php` command on every commit then it will be failing with error as mentioned in the above screenshot and we have to use `--no-verify` on most of the commits to bypass checks.

Let me know if it makes sense.

Reply:
This would need to work on _only_ the `includes` directory. Is that possible, @bradp?

Reply:
We can do that for sure, we should also put together the standards for the `src` directory so that we can get that in place at the same time.

@JasonTheAdams

Reply:
@JasonTheAdams I removed `npm run lint:php` command for now which prevents showing warnings when committing code instead of that it will format code and commit code.